pkg/logs/sender/destination.go

```diff
@@ -32,26 +33,45 @@ func (e *FramingError) Error() string {
 
 // Destination is responsible for shipping logs to a remote server over TCP.
 type Destination struct {
-	prefixer    Prefixer
-	delimiter   Delimiter
-	connManager *ConnectionManager
-	conn        net.Conn
+	prefixer  Prefixer
+	delimiter Delimiter
+	/* TODO: Merge ConnectionManager into Destination (think about GRPC integration before) */
```

Comment:
I think this is not likely to happen now and if this happens, we'll use core agent `Forwarder` I think

Reply:
removed